src/main/java/com/gachigang/ontherun/service/CategoryService.java

```diff
@@ -0,0 +1,40 @@
+package com.gachigang.ontherun.service;
+
+import com.gachigang.ontherun.persistence.entity.Category;
+import com.gachigang.ontherun.persistence.repository.CategoryRepository;
+import jakarta.persistence.EntityNotFoundException;
+import lombok.AllArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@AllArgsConstructor
```

Comment:
Change from @AllArgsConstructor to @RequiredArgsConstructor

Reply:
Replaced